lib/socket.ex

```diff
@@ -68,7 +68,7 @@ defmodule Channels.Client.Socket do
   you can simply return `{:stop, reason, state}` from any of the callback.
   """
   use GenServer
-  alias Channels.Client.WebsocketTransport
+  alias Phoenix.Channels.GenSocketClient.WebSocketClient
```

Comment:
[Phoenix uses WebSocket](https://hexdocs.pm/phoenix/Phoenix.Transports.WebSocket.html) so I think we should too. In the final version to `Websocket` so I think I should fix this.